platform/ios/src/MGLScaleBar.mm

```diff
@@ -102,6 +102,12 @@ @implementation MGLScaleBarLabel
 - (void)drawTextInRect:(CGRect)rect {
     CGSize shadowOffset = self.shadowOffset;
     UIColor *textColor = self.textColor;
```

Comment:
A simpler fix (that doesn’t touch iOS 13 API) would be to ignore `self.textColor`, delete this line, and then hardcode `UIColor.blackColor` at L121 where it was used.